src/System.Console/src/System.Console.csproj

```diff
@@ -138,6 +138,9 @@
     <Compile Include="$(CommonPath)\Interop\Unix\libcoreclr\Interop.GetFileInformation.cs">
       <Link>Common\Interop\Unix\Interop.GetFileInformation.cs"</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.IO.Native\Interop.NativeIO.cs">
```

Comment:
Not yet. I wanted to make the smallest possible change that would use the new shim. We still pull in SafeFileHandle from Common that I didn't refactor yet.